h2o-algos/src/main/java/water/userapi/Dataset.java

```diff
@@ -0,0 +1,207 @@
+package water.userapi;
+
+import water.DKV;
+import water.Iced;
+import water.Key;
+import water.Scope;
+import water.fvec.*;
+import water.parser.ParseDataset;
+import water.rapids.ast.prims.advmath.AstStratifiedSplit;
+import water.udf.specialized.Enums;
+import water.util.FileUtils;
+import water.util.FrameUtils;
+import water.util.VecUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.*;
+
+import static water.rapids.ast.prims.advmath.AstStratifiedSplit.*;
+/**
+ * Simplified Frame wrapper for simple usages
+ * 
+ * Created by vpatryshev on 2/20/17.
+ */
+public class Dataset  extends Iced<Dataset> {
```

Comment:
Frame is too heavy already, and it has too many details inside. Userapi is an adapter set of classes, with simple operations. If one wants a frame, a frame is available. But it's not a functionality of frame to deal at this level. Frame is too technical.